tasks/setup.py

```diff
@@ -204,6 +205,21 @@ def enable_pre_commit(ctx) -> SetupResult:
     return SetupResult("Enable pre-commit", status, message)
 
 
+def setup_vscode(ctx) -> SetupResult:
+    print(color_message("Setting up VS Code...", Color.BLUE))
+
+    status = Status.OK
+    message = ""
+
+    if yes_no_question("Do you want to set up VS Code?", default=True):
+        vscode.setup(ctx)
```

Comment:
I see some `assert` there which could stop the script. Can we catch the exception and return a `Status.KO` so the script can continue?